### PR TITLE
Update symfony from 5.3 to 5.4

### PR DESCRIPTION
**Pull request by xvilo**
Active support for Symfony 5.3 ended 6 days ago (01 Jan 2022). Symfony 5.4 was released 1 month and 1 week ago (29 Nov 2021) and it's support ends in 2 years and 10 months (30 Nov 2024).

### CODE REVIEW

**Comment by xvilo** [comment]
PHPUnit is ✅  was also able to do some stuff locally on my development machine. So I suspect this PR is fine 👍

File: tests/Functional/Command/ProxySyncReleasesCommandTest.php [hunk 1]
```diff
@@ -109,7 +108,7 @@ private function fakeLockFactory(): LockFactory
 
     private function lockFactory(): LockFactory
     {
-        /** @var Connection */
+        /** @var \PDO */
```
Comment (akondas) [inline tests/Functional/Command/ProxySyncReleasesCommandTest.php:111]: why not connection?

**Review by akondas — APPROVED** [review]

File: tests/Functional/Command/ProxySyncReleasesCommandTest.php (hunk 1 above)
Comment (xvilo) [reply tests/Functional/Command/ProxySyncReleasesCommandTest.php]: Good catch, this is was something I had to look into. The `PdoStore` is now `PDO` only, had to change this to `DoctrineDbalStore` instead and updated the `@var` doc comment back to `\Doctrine\DBAL\Connection`

**Comment by xvilo** [comment]
@akondas pushed the last change to address your comment, all should be fine now :)

**Comment by akondas** [comment]
Thanks @xvilo :beers:

**Pull request merged**